README.md

```diff
@@ -117,7 +116,9 @@ $ rails new . -d postgresql
 ```
 
 ```
-Your answer...
+The first terminal command will make a new rails directory called tunr, it will use the default database type (not the right word for it, but it will use sqlite and not postgresql)
+
+the second command will create a new rails directory, but it will be unamed, and will use postgresql as its database type.
 ```
```

Comment:
The second will not create a new Rails directory. It will actually place all those files and folders inside of your current directory.